### DISCUSSION TITLE
Video re-encoding

### DISCUSSION THREAD

**Discussion by kisst** [tfeldmann/organize#351] [Ideas]
Before I reinvent the wheel, I am looking for a rule which detect the video encoder used, and when it's suboptimal it re-encode it  with something decent, like av1 or h.265.

**Comment by tfeldmann** [comment]
I'm no expert in video encodings but wouldn't the quality suffer from a re-encoding?

**Comment by kisst** [comment]
Not really, think of it as you have a zip, you extract the content of it and then compress it back as a 7z, you will save space, and ffmpeg can do it just fine, the Q is about finding the files which can be re-encoded, and command invocation with the right parameters.

**Comment by tfeldmann** [comment]
How would you detect the current encoding?

**Comment by kisst** [comment]
ffprobe for example can do it, see AI generated [command](https://www.perplexity.ai/search/write-a-command-UFBrfpcASvyNOpt6gx0fTQ?s=u ) as reference ( untested )
